ra/ra.go

```diff
@@ -300,10 +300,12 @@ type ValidationProfileConfig struct {
 	// specified, the profile is open to all accounts. If the file
 	// exists but is empty, the profile is closed to all accounts.
 	AllowList string `validate:"omitempty"`
+	// IdentifierTypes is a list of identifier types that may be issued under
+	// this profile. If none are specified, it defaults to "dns".
+	IdentifierTypes []string `validate:"omitempty"`
```

Comment:
I think you can strengthen the validation here, like you did in the PA:
```suggestion
	IdentifierTypes []identifier.IdentifierType `validate:"omitempty,oneof=dns ip"`
```